pkg/crc/config/settings.go

```diff
@@ -59,7 +59,8 @@ func RegisterSettings(cfg *Config) {
 		fmt.Sprintf("Virtual machine preset (valid values are: %s, %s and %s)", preset.Podman, preset.OpenShift, preset.OKD))
 	// Start command settings in config
 	cfg.AddSetting(Bundle, defaultBundlePath(cfg), validateBundlePath, SuccessfullyApplied,
-		fmt.Sprintf("Bundle path (string, default '%s')", defaultBundlePath(cfg)))
+		fmt.Sprintf("Bundle path (string, like 'https://foo.com/%s', 'docker://quay.io/myorg/%s:%s' default '%s' )",
```

Comment:
It's called 'bundle path' and the example make it look as if it only accepts URIs :-/ Not sure it's worth having the example, 
`Bundle path/URI (absolute or local path, http, https or docker URI)`

Reply:
Added